src/realm/table.cpp

```diff
@@ -911,7 +911,7 @@ void Table::clear_indexes()
     }
 }
 
-void Table::do_add_search_index(ColKey col_key)
+void Table::do_add_search_index(ColKey col_key, IndexType type)
```

Comment:
This method needs to throw when adding a full text index to a column which is not a string column. (I saw full text code in this PR which uses Mixed::to_string() which will assert if the column type was UUID/ObjectID etc)

Reply:
:+1: